tests/api/test_result.py

```diff
@@ -49,19 +118,19 @@ def test_tdm_print(self, capfd):
         samples = np.ones((2, 3, 4))
         result = Result({"output": [samples]})
         print(result)
-        out, err = capfd.readouterr()
+        out, _ = capfd.readouterr()
         assert "spatial_modes=3" in out
         assert "shots=2" in out
         assert "timebins=4" in out
         assert "contains state=False" in out
 
-    def test_unkown_shape_print(self, capfd):
+    def test_unknown_shape_print(self, capfd):
         """Test that printing a result object with samples with an unknown shape
         provides the correct output."""
         samples = np.ones((2, 3, 4, 5))
         result = Result({"output": [samples]})
         print(result)
-        out, err = capfd.readouterr()
+        out, _ = capfd.readouterr()
```

Comment:
If you wanted to save yourself the throwaway `_`, I note that the `pytest` examples seem to recommend writing this as `captured = capfd.readouterr()` and making assertions against `captured.out`.

Reply:
Good idea! Updated!